Part-1-Functions

```diff
@@ -124,3 +148,12 @@ def fix_teen(n):
 
 def count_evens(nums):
   # CODE GOES HERE
+  lst=[1,2,3,4,5,6,8,10]
+count=0
+for i in lst:
+    if i%2==0:
+        
+        count=count+1
+    else:
+        i=i+1
+print(count)
```

Comment:
No need of i=i+1 just do count no need of else as we have to count only even numbers!